docs/java-rest/index.asciidoc

```diff
@@ -9,4 +11,6 @@ include::low-level/index.asciidoc[]
 
 include::high-level/index.asciidoc[]
 
+include::https://github.com/elastic/elasticsearch-java/docs/index.asciidoc[]
```

Comment:
Should this instead use something like:

```suggestion
include::{elasticsearch-java-root}docs/index.asciidoc[]
```

I'm not as familiar with how we do cross-book includes.

Reply:
@gtback Will `{elasticsearch-java-root}` resolve correctly? I don't see such an attribute.

Reply:
We use those in the ML guide, and they're talked about in https://github.com/elastic/docs#shared-attributes ("There is also a special set of attributes that are automatically created by the build process. They are labelled <repo>-root, where <repo> is the name defined at the top of the conf.yaml. For example, there is an elasticsearch-root attribute that resolves to the root path of the Elasticsearch repo. Please use these root attributes or define es-repo-dir, for example, rather than relying on intrinsic attributes..."

Reply:
I therefore think it needs to be this (extra slash compared to Greg's suggestion):
```suggestion
include::{elasticsearch-java-root}/docs/index.asciidoc[]
```